docs/api/rest_api/rest_api_reference/input/ibexa-types.raml

```diff
@@ -3614,6 +3614,34 @@ CartMetadataUpdate:
         names: Value[]
         descriptions: Value[]
 
+CartConstraintViolationListWrapper:
+    description: 'JSON object with only a CartConstraintViolationList property.'
+    type: object
+    properties:
+        CartConstraintViolationList: CartConstraintViolationList
+
+CartConstraintViolationList:
+    description: 'This class represents a list of cart constraint violations.'
+    type: object
+    properties:
+        violations:
+            type: array
+            items: CartConstraintViolation
```

Comment:
Maybe a CartConstraintViolationWrapper is needed but I didn't manage to product constraint violation to study it.